docs/api/public_php_api_search.md

```diff
@@ -10,7 +10,7 @@ This section covers how the [`SearchService`](http://apidoc.ez.no/sami/trunk/NS/
 
 Query object contains two properties you can set criteria on `filter` and `query`. You can mix and match use or use both at the same time, there is one distinction between the two:
 
--   `query` Has an effect on scoring *(relevancy)* calculation, and also on the default sorting if `sortClause` is not specified, *when used with Solr and Elastic.* Typically `query` is used for `FullText` search criterion, otherwise you can place everything else on `filter`.
+-   `query` Has an effect on scoring *(relevancy)* calculation, and also on the default sorting if `sortClause` is not specified, *when used with Solr.* Typically `query` is used for `FullText` search criterion, otherwise you can place everything else on `filter`.
```

Comment:
```suggestion
-   `query` — has an effect on scoring *(relevancy)* calculation and on the default sorting, if `sortClause` is not specified *when used with Solr.* Typically, `query` is used for `FullText` search criterion. Otherwise you can place everything else on `filter`.
```